src/main/java/com/hubspot/jinjava/lib/filter/DefaultFilter.java

```diff
@@ -75,7 +75,8 @@ public Object filter(
       );
     }
 
-    boolean truthy = (boolean) parsedArgs.get(TRUTHY_PARAM);
+    Object truthyObj = parsedArgs.get(TRUTHY_PARAM);
+    boolean truthy = truthyObj != null && (boolean) truthyObj;
```

Comment:
This may be simpler. What do you think?
```suggestion
    boolean truthy = Boolean.TRUE.equals(parsedArgs.get(TRUTHY_PARAM));
```